src/phasorpy/cluster.py

```diff
@@ -0,0 +1,193 @@
+"""Cluster phasor coordinates.
+
+The `phasorpy.cluster` module provides functions to:
+
+- cluster phasor coordinates using Gaussian Mixture Model (GMM):
+    - :py:func:`phasor_cluster_gmm`
+
+"""
+
+from __future__ import annotations
+
+__all__ = ['phasor_cluster_gmm']
+
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from ._typing import Any, ArrayLike, NDArray
+
+import math
+from collections.abc import Sequence
+
+import numpy
+from sklearn.mixture import GaussianMixture
+
+from ._utils import parse_skip_axis
+
+
+def phasor_cluster_gmm(
+    real: NDArray[numpy.float64],
+    imag: NDArray[numpy.float64],
+    sigma: float = 2.0,
+    /,
+    *,
+    clusters: int = 1,
+    skip_axis: int | Sequence[int] | None = None,
```

Comment:
I don't understand the purpose of `skip_axis` in the current implementation and suggest to remove it.

```suggestion
```